src/init.cpp

```diff
@@ -2000,6 +2000,13 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
         if (connect.size() != 1 || connect[0] != "0") {
             connOptions.m_specified_outgoing = connect;
         }
+        if (!connOptions.m_specified_outgoing.empty() && !connOptions.vSeedNodes.empty()) {
+            LogPrintf("-seednode is ignored in -connect mode\n");
+        }
+
+        if (args.GetBoolArg("-dnsseed", DEFAULT_DNSSEED) && args.IsArgSet("-proxy")) {
+            LogPrintf("-dnsseed is ignored in -connect mode when -proxy is specified\n");
```

Comment:
maybe s/in -connect mode when/when -connect is used and/

Reply:
Done